homeassistant/components/recorder/models.py

```diff
@@ -131,13 +131,9 @@ def entity_ids(self, point_in_time=None):
         Specify point_in_time if you want to know which existed at that point
         in time inside the run.
         """
-        from sqlalchemy.orm.session import Session
+        from homeassistant.components import recorder
```

Comment:
I want the `models.py` to be able to run without requiring the recorder component. That way we can use it in jupyter notebooks ([example](https://gist.github.com/balloob/dd05d872ac5b277c56e31d140b8d4d5c))